packages/landing/src/components/map.label.tsx

```diff
@@ -47,12 +47,22 @@ export class MapLabelControl implements IControl {
     Config.map.setLabels(labelState);
   };
 
+  /** Is the label button hidden from view */
+  isDisabled(): boolean {
+    // Most vector styles have overlaps with the labels and make them useless
+    if (Config.map.style && LabelsDisabledLayers.has(Config.map.style)) return true;
+    // Labels use the merge style feature and need the production configuration to work
+    if (Config.map.isDebug && Config.map.config) return true;
```

Comment:
NIL, `Config.map.config` this could point to production config as well. But it's ok.